examples/ode-integrator.dx

```diff
@@ -2,40 +2,56 @@
 Using only generic vector operations.
 
 Time : Type = Real
+def Eulerstate (a:Type) : Type = (a & Time & a & Time)
```

Comment:
docs pls? 😊

Reply:
Added an explanation - is this what you had in mind?

Reply:
Yeah, perfect. Thanks!